### DISCUSSION TITLE
Vil ikke beregne - General "Der skete en fejl..." bedsked

### DISCUSSION THREAD

**Discussion by stonani** [Eduap-com/WordMat#66] [Support] [answered]
Hej, 
Jeg har forsøget at opdatere Wordmat til 1.24.2, men den kan ikke fortage beregning, hverken med Maxima eller Geogebra som CAS motor (Den kan godt åbne en geogebra browser, men er mest interesseret i at den kan indsætte et resultat i Word uden at jeg behøver at kopiere resultatet ind efterfølgende). 

Med Maxima får jeg fejlbedskeden:
"Der skete en fejl. Wordmat er formentlig ikke installeret korrekt. Prøv at geninstallere evt. med Antivirus deaktiveret"
Efter jeg har trukket ok kommer en ny fejlbedsked fra Microsoft Visual Basic:
"Run-time error '-214722104 (80040154)':
Class not registered"

Med Geogebra får jeg blot den samme Microsoft Visual Basic fejbedskeden.

Jeg har selv gentagende gange prøvet at geninstallere Wordmat, og har i processen prøvet at afinstallere min antivirus Avast antivirus og slukke Windowsdefender "Virus & threat protection", men det løste ikke problemet. 
Jeg har herefter prøvet at give de forskellige .exe filer for Wordmat og Maxima ekstra tilladelse til Data Execution Prevention (DEP), da jeg fandt ud af det måske kunne være problemet, men det har heller ikke løst problemet.

Jeg bruger et Windows 11 system på en HP ENVY x360 og har en skole subscription til Microsoft 365 Apps. Word versionen er 2108.
Jeg har vedlagt billeder af fejlbedskederne og filen med den simple beregning (5+5), som ikke vil beregnes.
Nogen forslag til hvad fejlen kan være?
På forhånd mange tak :)

![image](https://user-images.githubusercontent.com/60426254/179401888-fdeb4baa-1443-44b3-b4ea-18376854eb98.png)

[Beregnings fejl.docx](https://github.com/Eduap-com/WordMat/files/9127797/Beregnings.fejl.docx)

**Comment by Eduap-com** [comment]
Mit bedste bud er at antivirus har været aktivt under installationen. Hvis det ikke er tilfældet ved jeg det ikke. Det vil nok kræve en nærmere undersøgelse af computeren.
Prøv at skifte til GeoGebra som CAS motor under indstillinger. Det vil formentlig virke.

**Comment by stonani** [comment]
Desværre har jeg prøvet uden antivirus og det var ikke tilfældet. (helt afinstalleret avast antivirus og slukke Windowsdefender "Virus & threat protection")
GeoGebra som CAS motor giver fejlen til venstre på billedet fra Visual Basic om at klassen ikke er registeret. 
Det er kun GeoGebra browser som fungerer, men det er ikke ideelt da jeg skal kopier svaret fra browseren.

**Comment by Eduap-com** [comment]
Det kunne tyde på fejl i Officeinstallationen. Prøv at geninstallere Office.

**Comment by stonani** [comment, marked as answer]
Det løste fejlen at geninstallere Office og derefter WordMat :)
Jeg brugte Microsoft office and support assistance: https://www.microsoft.com/en-us/download/100607, til at afinstallere office.

**Comment by Eduap-com** [comment]
Tak fordi skrev tilbage hvad der løste fejlen. Det vil andre der ser beskeden sætte pris på.
Jeg flytter dette issue til discussions.
